### PR TITLE
feat: update entry

### LINKED ISSUE #333: feat: update entry

### PR DESCRIPTION
**Pull request by vzart**

### CODE REVIEW

File: src/Application/Digital/Commands/UpdateEntry.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                         && x.EmployeeId == request.CurrentUserId, cancellationToken);
+
+                if (permission is null ||
+                    !permission.AllowedOperations
+                        .Split(",")
+                        .Contains(EntryOperation.Upload.ToString()))
+                {
+                    throw new UnauthorizedAccessException("User cannot access this resource.");
+                }
+            }
```
Comment (kaitoz11) [inline src/Application/Digital/Commands/UpdateEntry.cs]: This code kinda smell....
Delete line `46-49`?

**Review by kaitoz11 — CHANGES_REQUESTED** [review]
Small shit that you may missed?

File: src/Application/Digital/Commands/UpdateEntry.cs (hunk 1 above)
Comment (vzart) [reply src/Application/Digital/Commands/UpdateEntry.cs]: oh my god, I thought I deleted it, I deserve to die, I will hang myself now

**Review by kaitoz11 — APPROVED** [review]
LGTM

**Pull request merged**